bears/python/requirements/PySafetyBear.py

```diff
@@ -74,12 +74,20 @@ def run(self, filename, file):
                 version_spec_match.end(1) + 1,
             )
 
-            yield Result(
-                self,
-                message_template.format(vuln=vulnerability),
-                additional_info=vulnerability.description,
-                affected_code=(source_range, ),
-            )
+            try:
+                yield Result(
+                    self,
+                    message_template.format(vuln=vulnerability),
+                    additional_info=vulnerability.description,
+                    affected_code=(source_range, ),
+                )
+            except KeyError:  # Ignore PycodestyleBear (E722)
```

Comment:
You can remove this comment now since you have specified the exception name, it will now conform to standards :)
I had a look at your circleCI build, I think you should have a look at other coala-bears PRs that were submitted to see if this issue is coming up only for this or others too :)